package.json

```diff
@@ -94,7 +94,7 @@
     "lint": "eslint -c .eslintrc.js --ext .ts src",
     "civersion": "node ci/version.js",
     "prepare": "npm run build",
-    "preinstall": "npm install --package-lock-only --ignore-scripts --no-audit && npx npm-force-resolutions"
+    "preinstall": "npm install --package-lock-only --ignore-scripts --no-audit"
```

Comment:
Please read https://github.com/microsoft/cognitive-services-speech-sdk-js/issues/549#issuecomment-1300262225
Running `npm install` in the `preinstall` script here will run it when a user installs this package messing up their node_modules. This also does not work with yarn or pnpm.
Use a alternate script name like `setup` to allow your developers to run this script.

Reply:
@glharper @BrianMouncer